README.md

```diff
@@ -67,10 +67,16 @@ Apache Polaris supports the following optional build options:
 ### More build and run options
 Running in Docker
 - `docker build -t localhost:5001/polaris:latest .` - To build the image.
+  - Optional build options:
+    - `docker build -t localhost:5001/polaris:latest --build-arg ECLIPSELINK=true .` - Enables the EclipseLink extension.
+    - `docker build -t localhost:5001/polaris:latest --build-arg ECLIPSELINK=true --build-arg ECLIPSELINK_DEPS=[groupId]:[artifactId]:[version],... .` – Enabled the EclipseLink extension with one or more additional dependencies for EclipseLink (e.g., JDBC drivers), separated by commas.
 - `docker run -p 8181:8181 localhost:5001/polaris:latest` - To run the image in standalone mode.
 
 Running in Kubernetes
 - `./run.sh` - To run Polaris as a mini-deployment locally. This will create one pod that bind itself to ports `8181` and `8182`.
+  - Optional run options:
+    - `./run.sh -b "ECLIPSELINK=true"` - Enables the EclipseLink extension.
+    - `./run.sh -b "ECLIPSELINK=true;ECLIPSELINK_DEPS=[groupId]:[artifactId]:[version],..."` – Enabled the EclipseLink extension with one or more additional dependencies for EclipseLink (e.g., JDBC drivers), separated by commas.
```

Comment:
It's semicolons not commas right?

Reply:
The deps for `ECLIPSELINK_DEPS` but the diff options are separated by commas. So following format:
```
-b "KEY1=V1;KEY2=V2,V3,V4;KEY3=v4"
```

This is because the delimiter is comma for diff optional options. Thus, I can't use comma to split deps as well.